app/code/Mykhailok/AskAboutThisProduct/etc/adminhtml/system.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<config xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:module:Magento_Config:etc/system_file.xsd">
+    <system>
+        <section id="trans_email" translate="label" type="text" sortOrder="90" showInDefault="1" showInWebsite="1" showInStore="1">
```

Comment:
@mykhailok-dv Invalid implementation. There is no need to add new email recipient unless requested. This is not the case. Must implement ability to select a recipient from the 5 available variants like this is done for order emails and other emails in the system.